student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom)
+    super
```

Comment:
- [ ] I think here you missed some parameters so here when inheriting a class so also it inherit the paramters so please add the parameters for age, name and parent_permission with their defult values 💯🔥💪🏻😊


- [ ] As I can see also here you missed to pass the super() parameters because you inheirt from a class so also you should pass the parameters so please add it and pass base class parameters like above in student class 💪🏻🔥